src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,10 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
-    public String getRandomColor() {
-        return null;
+    public Color getRandomColor() {
+        int index = new Random().nextInt(Color.values().length);
```

Comment:
move Random to the class field

Reply:
not fixed
![image](https://github.com/mate-academy/jv-lottery/assets/20326858/8af8b8c4-e968-41b0-a56d-67d9c1491bc7)